invoice.py

```diff
@@ -756,6 +810,7 @@ def transition_generate(self):
         Invoice = Pool().get('account.invoice')
 
         invoices = Invoice.browse(Transaction().context['active_ids'])
-        service = 'generate_facturae_%s' % self.start.service
-        getattr(Invoice, service)(invoices, self.start.certificate_password)
+        for invoice in invoices:
```

Comment:
A wizard is no longer necessary. A simple button to regenerate in case the worker failed and that gets the service from the configuration is enough.

Reply:
Wizard is required to select the service to send (pimec, etc) or select "none" in case user don't like to send (only generate and sing file)

Reply:
I'm not really sure there's a use case for that, but ok for now.